enclave_wrangler/objects_api.py

```diff
@@ -351,60 +348,33 @@ def fetch_cset_and_member_objects(
-        # maybe log looks like
-        #   audit table columns:
-        #       table_name: code_sets | concept_set_container | concept_set_members | concept_set_version_item | researchers? etc.
-        #       primary_key: codeset_id, concept_set_name, [codeset_id, concept_id] | [codeset_id, concept_id] | ...
-        #       key_values:     ....
-        #       status:  success | failed from too many | 0 members, keep trying | failure other reason |
-        #           0 members after a couple hours = success
-        #           initial and final status?
-        #       comment text[]?
-        #       fixed_later: false (change to true later?)
-        #       processed: timestamp, initially null # used by code that updates the derived tables
-
         version: int = cset['properties']['codesetId']
-        # Hierarchical
-        # TODO: exc handling: Will need to handle differently or do more: https://github.com/jhu-bids/TermHub/issues/451
         try:
             cset['expression_items']: List[Dict] = get_concept_set_version_expression_items(version, return_detail='full')
         except EnclavePaginationLimitErr as err:
             cset['expression_items']: List[Dict] = err.args[1]['results_prior_to_error']
-            # TODO: flag in some table that we need to do datasets api download
-            # cset['termhub-notes']['expression_items_failed_after'] = len(cset['expression_items'])
-            print(pagination_err.format(version), file=sys.stderr)
-
+            fetch_statuses.append({'table': 'code_sets', 'primary_key': version, 'status_initially':
+                'fail-excessive-items', 'comment': f"Failed after {len(cset['expression_items'])} expression_items."})
         try:
             cset['member_items']: List[Dict] = get_concept_set_version_members(version, return_detail='full')
-            auditMsg = f"found {len(cset['member_items'])} members"
         except EnclavePaginationLimitErr as err:
             cset['member_items']: List[Dict] = err.args[1]['results_prior_to_error']
-            print(pagination_err.format(version), file=sys.stderr)
-            auditMsg = f"failed after {len(cset['member_items'])} members"
-
+            fetch_statuses.append({'table': 'code_sets', 'primary_key': version, 'status_initially':
+                'fail-excessive-members', 'comment': f"Failed after {len(cset['member_items'])} members."})
         if not cset['member_items']:
-            auditMsg = f"got no members as of {str(datetime.now())}, will continue trying until..."
-            pass
-            # TODO: do something so we keep trying for a couple hours. it's possible it really doesn't have members, but
-            #   probably the members just aren't available yet
-
-
+            fetch_statuses.append({'table': 'code_sets', 'primary_key': version, 'status_initially': 'fail-0-members',
+                'comment': f"Fetched 0 members after fetching {len(cset['expression_items'])} items."})
```

Comment:
FYI: What it looks like when reporting failure to fetch any members (also shows how many expression items were fetched).